index.html

```diff
@@ -15,7 +15,11 @@
 <body>
     <header>
         <h1><a class="logo" href="#">My Logo</a></h1>
-        <nav class="menu"><span class="material-icons">menu</span></nav>
+        <div class="menu" tabindex="2" id="mobileMenuButton">
+            <div class="toogle-btn"></div>
+            <div class="toogle-btn"></div>
+            <div class="toogle-btn"></div>
+        </div>
```

Comment:
- [ ] On desktop size the hamburger menu button should not be visible, currently your version is not behaving like this, would you address it so that it will increase user experience while using it ?